manager/src/main/java/io/gingersnapproject/database/vendor/VendorUtil.java

```diff
@@ -0,0 +1,25 @@
+package io.gingersnapproject.database.vendor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.gingersnapproject.database.model.PrimaryKey;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+
+/**
+ * TODO!
+ */
+public enum VendorUtil {
+   ;
+
+   public static PrimaryKey extracPrimaryKey(RowSet<Row> rows) {
```

Comment:
I know this is a copy+paste, but there's a type :slightly_smiling_face:  `extractPrimaryKey`